src/app/features/tech-record/components/test-record-summary/test-record-summary.component.ts

```diff
@@ -20,6 +21,9 @@ export class TestRecordSummaryComponent {
   @Input() testRecords: TestResultModel[] = [];
   @Input() currentTechRecord?: TechRecordModel;
 
+  public get Roles() {
```

Comment:
Other component have a lowercase `roles` getter (just to keep consistent)

Reply:
done  :)

Reply:
Ah turns out I had turned on Match case on the search... 😆